### PR TITLE
PMP-1793 | JAN-614

### PR DESCRIPTION
**Pull request by dtiwarATS**
-- updated contains operator logic

### CODE REVIEW

File: assets/src/adaptivity/operators/contains.ts [hunk 1]
```diff
@@ -36,11 +36,16 @@ const handleContainsOperator = (factValue: any, value: any, isDoesNotContainsOpe
...
       );
     } else {
-      return (
-        modifideValue
-          // check if value is found in factValue array
-          .some((item) => updatedFacts.includes(item))
-      );
+      let hitCount = 0; // counts the number of values found
+
+      modifideValue.forEach((item) => {
```
Comment (bsparks) [inline assets/src/adaptivity/operators/contains.ts:41]: `modified` (typo)

File: assets/src/adaptivity/operators/contains.ts [hunk 2]
```diff
@@ -36,11 +36,16 @@ const handleContainsOperator = (factValue: any, value: any, isDoesNotContainsOpe
...
+        if (updatedFacts.includes(item)) {
+          hitCount++;
+        }
+      });
+      if (hitCount == modifideValue.length) {
```
Comment (bsparks) [inline assets/src/adaptivity/operators/contains.ts:46]: should use `===`

File: assets/src/adaptivity/operators/contains.ts (hunk 1 above)
Comment (dtiwarATS) [reply assets/src/adaptivity/operators/contains.ts]: Fixed

File: assets/src/adaptivity/operators/contains.ts (hunk 2 above)
Comment (dtiwarATS) [reply assets/src/adaptivity/operators/contains.ts]: Fixed

**Review by bsparks — APPROVED** [review]

**Pull request merged**